examples/dynamics/thin_rod_as_conical_pendulum.py

```diff
@@ -53,8 +54,8 @@
 )
 
 # Let us consider a non-inertial reference frame in which the rod is at rest, i.e. it is in equilibrium.
-# The `z` axis is codirectional to the vector of angular velocity and the `y` axis lies in the plane where
-# the rod is oscillating.
+# The `z` axis is codirectional to the vector of angular velocity and the `y` axis is parallel to the radial
+# axis if a cylindrical coordinate system is assumed.
```

Comment:
Well, if Z axis is collinear to angular velocity, Y is obviously orthogonal to it, so it should be parallel to radial plane. So you description of Y axis is useless. I suggest to describe origin point instead.